src/components/demo/demo-creator.tsx

```diff
@@ -7,7 +7,7 @@ import { parseUrl, stringify } from "query-string";
 
 import "./demo-creator.sass";
 
-export const NUM_FAKE_CLASSES = 9;
+export const NUM_FAKE_CLASSES = 100;
```

Comment:
This change seems unrelated to the other changes. Is there a particular reason for it?